client/src/app/components/login-form/login-form.component.html

```diff
@@ -0,0 +1,38 @@
+<form (ngSubmit)="onSubmit()" class="login-form">
```

Comment:
```suggestion
<form (ngSubmit)="onSubmit()">
```
If there is no styling necessary, we can leave these off.